cmd/generate/cmd_util.go

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/nginxinc/nginx-go-crossplane/internal/generator"
+)
+
+type filterFlag struct {
+	filter map[string]struct{}
+}
+
+func (reader *filterFlag) Set(value string) error {
+	if reader.filter == nil {
+		reader.filter = make(map[string]struct{})
+	}
+	reader.filter[value] = struct{}{}
+	return nil
+}
+
+func (reader *filterFlag) String() string {
+	return fmt.Sprintf("%v", *reader)
+}
+
+type overrideItem struct {
+	directive string
+	masks     []generator.Mask
+}
+
+func (item *overrideItem) UnmarshalText(text []byte) error {
+	rawOverride := string(text)
+
+	// rawStr should follow the format: directive:bitmask00|bitmask01|...,bitmask10|bitmask11|...
+	directive, definition, found := strings.Cut(rawOverride, ":")
+	if !found {
+		return fmt.Errorf("colon not found, override=%s, ", rawOverride)
```

Comment:
I'm not sure why you've added an extra comma and space?

```suggestion
		return fmt.Errorf("colon not found, override=%s", rawOverride)
```

If you're concerned that printing the override could look ambiguous, you could use `%q` to wrap the string in quotes.

Reply:
https://github.com/nginxinc/nginx-go-crossplane/pull/106#discussion_r1681359367. Sorry for omitting removal of rawOverride output here. Just removed